client/python/tests/test_composite.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2018-2024 contributors to the OpenLineage project
+# SPDX-License-Identifier: Apache-2.0
+from __future__ import annotations
+
+from unittest import mock
+from unittest.mock import MagicMock
+
+from openlineage.client.transport.composite import CompositeConfig, CompositeTransport
+from openlineage.client.transport.transport import Transport
+
+
+def test_composite_loads_full_config() -> None:
+    config = CompositeConfig.from_dict(
+        {
+            "type": "composite",
+            "transports": {
+                "kafka": {
```

Comment:
Can we move transport name to key inside transport, rather than just being a key tied to `CompositeTransport`?

Reply:
I know it's for easier configuration through env vars but maybe let's add it in the coming PR related to this? (I'm assuming this PR merged -> next PR with env vars).